src/styles/styles.css

```diff
@@ -1,5 +1,292 @@
+  color: var(--white-two);
+}
+
+.active {
+  opacity: 1;
+}
+
+.store {
+  padding-left: 3rem;
+  font-family: var(--Montserrat);
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: var(--azure);
+}
+
+.card {
+  margin: 2rem auto;
+  width: 85%;
+  padding: 2rem 4rem 1.625rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.book-category {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: var(--Montserrat);
+  font-size: 1.2rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.book-title {
+  margin: 0;
+  font-size: 1.7rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  font-family: var(--RobotoSlab);
+}
+
+.book-author {
+  margin: 0;
+  font-family: var(--RobotoSlab);
+  font-size: 1.2rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  padding-left: 0;
+  align-items: center;
+}
+
+.comment-btn,
+.remove-btn,
+.edit-btn {
+  font-family: var(--RobotoSlab);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+  border: none;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+.book-card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-info {
+  width: 40%;
+}
+
+.book-info p {
+  padding-left: 7px;
+}
+
+.item-progress {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  font-family: var(--Montserrat);
+}
+
+.main-levels {
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.circle {
+  margin-right: 10%;
+}
+
+.main-task {
+  font-family: var(--RobotoSlab);
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.Update-progress {
+  font-family: var(--RobotoSlab);
+  padding: 0.8em 1.4rem;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+  border-radius: 3px;
+  background-color: var(--azure);
+  border: none;
+}
+
+.Line {
+  width: 90%;
+  border: solid 1px var(--white);
+  margin: 0 auto;
+}
+
+.form-container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.form-title {
+  font-family: var(--Montserrat);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
```

Comment:
Original design
![bookstore](https://github.com/mohammadnewton/bookstore/assets/78251927/f6f89352-3a50-402c-ab35-51234c4e8603)

Your design

To ensure our final design matches Zeplin's design, please add spacing  (right and left) of the form 🛠👍

![image](https://github.com/mohammadnewton/bookstore/assets/78251927/32c88cde-821d-4eed-9211-5c52dbd0ebff)

Your design
![image](https://github.com/mohammadnewton/bookstore/assets/78251927/f8c872e0-e3a0-4acc-8713-61b9f2ca3dfd)

Design on Zeplin
![image](https://github.com/mohammadnewton/bookstore/assets/78251927/fd998c7e-144c-4a93-a6d1-6701c9270c27)


Kindly use the same font style and text color according to the given design 🤗